DESCRIPTION

```diff
@@ -1,48 +1,39 @@
-Package: PACTA.analysis
-Title: Helpers for 'PACTA_analysis'
+Package: PACTA.analysis.dependencies
```

Comment:
Ultra-Blocking:
🤦 I think I know where I miscommunicated, that led you to think this is the appropriate method. I mentioned in one of the other PRs that I prefer using DESCRIPTION to manage dependencies over the other methods. What I failed to follow up with is "...as part of a full migration to an R package". 

Using this mechanism in this context is hacky, and the entire point of this exercise is to reduce our tech debt. Going with this approach instead is giving us a partial conversion to a package, but missing out on most of the benefits.

I would rather have our dependencies defined in a script or in the dockerfile. We can chaat later about what that looks like, but I'm imagining something along the lines of:

```r
pacta_dependencies <- c(
  "foo",
  "bar",
  "bax"
)

install.packages(pacta_dependencies)
```

and in the dockerfile:

```
COPY  install_deps.R .
RUN [Rscript, "install_deps.R"]
```

Reply:
A slightly different version that would give us better version control might be something along the lines of:
```r
packages <- rbind(
  c("foo", "12.3"),
  c("Bar", ">= 1.2.3"),
  c("baz", "1.5")
)
for (i in nrow(packages)){
  remotes::install_version(package = packages[i,1], version = packages[i,2])
}
```